pkg/etw/eventmetadata.go

```diff
@@ -10,6 +12,8 @@ type inType byte
 
 // Various inType definitions for TraceLogging. These must match the definitions
 // found in TraceLoggingProvider.h in the Windows SDK.
+//
+//nolint:deadcode,varcheck // keep unused constants for potential future use
```

Comment:
is this going to make it to the docstrings?

Reply:
nope, i think anything without a leading space is omitted